homeassistant/components/tuya/vacuum.py

```diff
@@ -204,4 +204,4 @@ def send_command(self, command: str, params: str = None, **kwargs: Any) -> None:
         """Send raw command."""
         if params is None:
```

Comment:
To guard against an emtpy list:

```suggestion
        if not params:
```